dashboard/src/app/workspaces/create-workspace/devfile-selector/devfile-selector.controller.ts

```diff
@@ -57,11 +70,11 @@ export class DevfileSelectorController {
 
     let location = this.cheWorkspace.getWorkspaceSettings().cheWorkspaceDevfileRegistryUrl;
```

Comment:
We don't need this variable more.

As @sleshchenko wrote:
cheWorkspaceDevfileRegistryUrl -> cheWorkspaceDevfileRegistryUrls

Reply:
> cheWorkspaceDevfileRegistryUrl -> cheWorkspaceDevfileRegistryUrls

This change is not done on Che Server side, so it may be OK to leave as is for time being.